api/src/main/java/javax/enterprise/inject/spi/ProcessAnnotatedType.java

```diff
@@ -42,10 +45,11 @@
  * </p>
  * 
  * @author David Allen
+ * @author Antoine Sabot-Durand
  * @see AnnotatedType
  * @param <X> The class being annotated
  */
-public interface ProcessAnnotatedType<X> {
+public interface ProcessAnnotatedType<X> extends AnnotatedTypeBuilderProvider {
```

Comment:
What will happen in the following case?:
` void observe(@Observes AnnotatedTypeBuilderProvider event)`
ISE or is it ok?